packages/features/ee/organizations/lib/server/createOrganizationFromOnboarding.ts

```diff
@@ -441,6 +458,7 @@ async function handleOrganizationCreation({
   // Connect the organization to the onboarding
   await OrganizationOnboardingRepository.update(organizationOnboarding.id, {
     organizationId: organization.id,
+    isComplete: true,
```

Comment:
We shouldn't mark the onboarding as complete here. As there are steps like inviting members and moving teams to org later. 

Removing it.